services/reference/bnb-smart-chain/quickstart.md

```diff
@@ -15,7 +15,15 @@ Don't have an Infura account? Sign up for our free plan and start using the BNB
 
 ## Prerequisites
 
-Ensure you have an [API key](../../../developer-tools/dashboard/get-started/create-api) with the BNB Smart Chain networked enabled.
+- Ensure you have an [API key](../../../developer-tools/dashboard/get-started/create-api) with the BNB Smart Chain networked enabled.
+
+- For calls that require gas, topup your funds.
+
+:::note Get testnet ETH
+
+To topup your [MetaMask](https://metamask.io/) wallet, refer to the [official BNB Smart Chain documentation](https://docs.bnbchain.org/bnb-smart-chain/developers/faucet/)
+and learn about faucets for BNB Smart Chain.
+:::
```

Comment:
```suggestion
```

This is not needed in the MetaMask docs because we have a how-to page on testnet tokens, and MM developers are more aware of how to acquire testnet funds.

Reply:
Same suggestion --> index See Also, change to link to how-to page